Example/Example/EditorDemoController.swift

```diff
@@ -244,29 +230,12 @@ class EditorDemoController: UIViewController {
     // MARK: - Title and Title placeholder position methods
     func updateTitlePosition() {
         let referenceView: UIScrollView = editingMode == .richText ? richTextView : htmlTextView
-        titleTopConstraint.constant = -(referenceView.contentOffset.y+referenceView.contentInset.top)
+        titleTopConstraint.constant = -(referenceView.contentOffset.y + referenceView.contentInset.top) + 5
 
         var contentInset = referenceView.contentInset
-        contentInset.top = (titleHeightConstraint.constant + separatorView.frame.height)
+        contentInset.top = titleHeightConstraint.constant - 10
```

Comment:
Mind replacing the `-10` with, perhaps... `(Constants.titleInsets.top + Constants.titleInsets.bottom)` (or so!)